count-cysticFibrosis.rq

```diff
@@ -0,0 +1,22 @@
+#+ endpoint: http://ejprd.fair-dtls.surf-hosted.nl:8085/blazegraph/namespace/kb/sparql
```

Comment:
@NuriaQueralt why do you name this query `count-cysticFibrosis`? Looks like this query return count for any resources right?

Reply:
@rajaram5 thanks for detecting it! Absolutely it is a mistake. This query counts the number of themes by biobank/patient registry. I will change the name.